emails/tests/mgmt_send_welcome_emails.py

```diff
@@ -0,0 +1,100 @@
+import pytest
+from typing import Tuple
+from unittest.mock import MagicMock, patch
+
+from django.conf import settings
+from django.contrib.auth.models import User
+from django.core.management import call_command
+
+from allauth.socialaccount.models import SocialAccount
+from django_ftl import activate
+
+from emails.models import Profile
+from emails.tests.models_tests import make_free_test_user
+from privaterelay.ftl_bundles import main as ftl_bundle
+
+
+COMMAND_NAME = "send_welcome_emails"
+
+
+@pytest.fixture()
+def mock_ses_client():
+    with patch("emails.apps.EmailsConfig.ses_client") as mock_ses_client:
+        yield mock_ses_client
+
+
+@pytest.mark.django_db
+def test_no_profiles_need_welcome_email(caplog: pytest.LogCaptureFixture):
+    profiles_without_welcome_email = Profile.objects.filter(sent_welcome_email=False)
+    assert profiles_without_welcome_email.count() == 0
+    call_command(COMMAND_NAME)
+    rec1, rec2, rec3 = caplog.records
+    assert "Starting" in rec1.getMessage()
+    assert rec2.getMessage() == "Emails to send: 0"
+    assert "Exiting" in rec3.getMessage()
+
+
+@pytest.mark.django_db
+def test_no_locale_defaults_to_en(
+    mock_ses_client: MagicMock, caplog: pytest.LogCaptureFixture
+):
+    user = _make_user_who_needs_welcome_email_with_locale("")
+
+    call_command(COMMAND_NAME)
+    _assert_caplog_for_1_email_to_user(user, caplog)
+
+    to_addresses, source, body_html = _get_send_email_args(mock_ses_client)
+    assert to_addresses == [user.email]
+    assert source == settings.RELAY_FROM_ADDRESS
+    assert 'lang="en"' in body_html
+
+
+@pytest.mark.django_db
+@pytest.mark.parametrize("locale", ("en-US,en;q=0.5", "de;q=0.7,en;q=0.3"))
+def test_send_welcome_emails(
+    locale: str, mock_ses_client: MagicMock, caplog: pytest.LogCaptureFixture
+):
+    ftl_bundle.reload()
+    user = _make_user_who_needs_welcome_email_with_locale(locale)
+    activate(user.profile.language)
```

Comment:
This causes later tests to break, when `de` is left as the default language. The safer way is to wrap in a context:

```
with django_ftl.override(user.profile.language):
    expected_cta = ftl_bundle.format("first-time-user-email-hero-cta")
# do some stuff
assert expected_cta in body_html
```